code/modules/urist/antagonist/revenant/archetypes/vampiric.dm

```diff
@@ -372,7 +372,7 @@
 
 /* FANGS - gives mobs a new unarmed attack that requires head and ignores restraints. */
 /datum/unarmed_attack/bsrevenant_vampbite
-	attack_verb = list("bit", "sank his fangs in")
+	attack_verb = list("bit", "sank their fangs in")
```

Comment:
you can use the /his macro here

Reply:
Ah, cool, wasn't sure when it resolves and how

Reply:
Done